pwa/src/templates/templateParts/resultsTemplates/table/TableResultTemplate.tsx

```diff
@@ -69,7 +71,7 @@ export const TableResultTemplate: React.FC<TableResultTemplateProps> = ({ compon
   };
 
   return (
-    <TableWrapper>
+    <TableWrapper touchScreen={screenSize === "tablet" || screenSize === "mobile"}>
```

Comment:
See my comment on the `TableWrapper` itself in the conduction/components pull request.

Would be nice if we could simply implement this like:

```
<HorizontalOverflowWrapper>
 ... content
</Horizon...>
```